### DISCUSSION TITLE
%E5%85%A5%E5%AD%A6%E7%AF%87/%E5%BF%AB%E9%97%AE%E5%BF%AB%E7%AD%94

### DISCUSSION THREAD

**Discussion by giscus** [SurviveXJTU/SurviveXJTU.github.io#5] [Announcements]
# %E5%85%A5%E5%AD%A6%E7%AF%87/%E5%BF%AB%E9%97%AE%E5%BF%AB%E7%AD%94

西安交大生存指南，更适合西交大学子的生存指南。

https://survivexjtu.github.io/%E5%85%A5%E5%AD%A6%E7%AF%87/%E5%BF%AB%E9%97%AE%E5%BF%AB%E7%AD%94.html

**Comment by Askiike** [comment]
军训手机管的不严，基本人人带手机，休息的时候可以看，但不要太明目张胆，有点眼色就行，不要在领导视察的时候掏出来

**Comment by weaver618** [comment]
兴庆这边洗澡的话，对全体开放的只有温泉浴室和崇实浴室哦
